### PR TITLE
Update nth-check and postcss dependencies

### PR DESCRIPTION
**Pull request by HarHarLinks**
**:heavy_check_mark: Checklist**

- [x] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [ ] Added or updated documentation.
- [ ] Tests for new functionality and regression tests for bug fixes.
- [ ] Screenshots or videos attached (for UI changes).
- [x] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,2 +1 @@
-#!/usr/bin/env sh
```
Comment (weeman1337) [inline .husky/pre-commit:1]: This looks like an unrelated change.

File: .changeset/five-numbers-buy.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@nordeck/matrix-neoboard-widget': patch
+'@nordeck/matrix-neoboard-react-sdk': patch
+---
+
+bump dev dependencies
```
Comment (weeman1337) [inline .changeset/five-numbers-buy.md:6]: We usually don't put dependency updated to the changelog.

File: yarn.lock [hunk 3]
```diff
@@ -21,447 +21,351 @@
     "@jridgewell/trace-mapping" "^0.3.24"
 
 "@apideck/better-ajv-errors@^0.3.1":
-  version "0.3.3"
```
Comment (weeman1337) [inline yarn.lock:24]: Where do all the changes come from? Is this related to the postcss update? It also looks like, that it includes other changes. Can you try to only include the necessary changes for updating postcss and nth-check?

**Review by weeman1337 — CHANGES_REQUESTED** [review]
May need some changes. Have a look at the comments.

File: yarn.lock (hunk 3 above)
Comment (HarHarLinks) [reply yarn.lock:24]: comes from indirectly updating postcss by actually updating resolve-url-loader

File: .changeset/five-numbers-buy.md (hunk 2 above)
Comment (HarHarLinks) [reply .changeset/five-numbers-buy.md]: done

File: yarn.lock (hunk 3 above)
Comment (weeman1337) [reply yarn.lock:24]: When I apply the same resolution, it changes only a few dependencies. Please re-check

**Review by weeman1337 — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 3 above)
Comment (HarHarLinks) [reply yarn.lock]: mondays...

**Review by weeman1337 — APPROVED** [review]
Looks good now :+1:
